src/main/java/uk/gov/hmcts/reform/preapi/repositories/RecordingRepository.java

```diff
@@ -98,17 +93,6 @@ Page<Recording> searchAllBy(
 
     boolean existsByCaptureSessionAndDeletedAtIsNull(CaptureSession captureSession);
 
-    @Query("""
-        update #{#entityName} e
-        set e.deletedAt=CURRENT_TIMESTAMP
-        where e.captureSession=:captureSession
-        and e.deletedAt is null
-        """
-    )
-    @Modifying
-    @Transactional
-    void deleteAllByCaptureSession(CaptureSession captureSession);
-
```

Comment:
This doesn't seem relevant to the PR - is this just tidying up as you go along or has this been removed in error?

Reply:
This is just tidying up unused cod